teku/src/main/java/tech/pegasys/teku/cli/options/ZkChainOptions.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Consensys Software Inc., 2025
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.cli.options;
+
+import picocli.CommandLine;
+import tech.pegasys.teku.config.TekuConfiguration;
+import tech.pegasys.teku.services.zkchain.ZkChainConfiguration;
+
+public class ZkChainOptions {
```

Comment:
We missed the unit tests for this options object. For example, testing that both boolean parameters default to false, that we can't have negative values on the integer parameter, etc. Otherwise, anyone could come here and change it by mistake, and we won't catch it.

Reply:
added them